docs/backend/setup/database-migrations.md

```diff
@@ -158,6 +159,13 @@ Migrations are run by the `milmove migrate` command. This allows us to leverage
 
 The reason to use a `make` target is because it will correctly set the migration flag variables and target the correct database with environment variables.
 
+## Update Migrations Locally
+In the event that you need to make an edit to a migration that you have just created prior to merging it into the main branch, 
+you can update the migration with your edits and rerun it using: `make db_dev_reset db_dev_migrate` 
```

Comment:
> Minor suggestion, this makes it so there's a "copy" button on the right so people can quickly copy the command. Kind of like the first command in [the Creating Secure Migration section](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations/#creating-secure-migrations). Contrasted with the commands in [the Running Migrations section](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations/#running-migrations). See how in the latter, they're formatted, but have to be highlighted to be copied, while in the former, there's a copy button on the right when you hover over the box.

Thanks @felipe-lee , do you happen to understand the difference between writing shell and writing console? I see we do both in this file.

Reply:
I'm not sure what the difference is tbh. I tried looking into it and Docusaurus has this section on supported languages:
 https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages

Based on that, it looks like the default supported languages are these:
https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js

But neither `shell` nor `console` are on there...You can extend the default list, but looking at our docusaurus config, it looks like we haven't added more languages: https://github.com/transcom/mymove-docs/blob/071f511b101d71df7aeabe893fb1a7a1191c946c/docusaurus.config.js#L149-L152

Digging into it some more, it looks like this is the full list of supported languages: https://prismjs.com/#supported-languages

That docs says `shell` is an alias for `bash` so I guess that's why that one works, but there's no mention of `console`

I tried looking into it some more, and ended up on [a thread with people talking about the syntax highlighters github](https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown-files) uses (or people think they use?). That pointed me to these two different highlighters:

* linguist
    * `console`: https://github.com/github/linguist/blob/3c3b037910006fc2f1a9bb34b2c4e9cde062206c/lib/linguist/languages.yml#L5837-L5848
    * `shell`: https://github.com/github/linguist/blob/3c3b037910006fc2f1a9bb34b2c4e9cde062206c/lib/linguist/languages.yml#L5753-L5824
    * They seem to be mostly the same except `shell` defines file names and one of the attributes, but I don't know enough about this to know what that difference means.
* highlight.js
    * https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md
    * This one treats `shell` and `console` the same way it seems.

So yeah, I'm not entirely sure. Based on the thread I linked above where people were talking about the gh syntax one, it seems `console` might be able to handle the terminal prompt, but I haven't really tested it. I personally lean toward `shell` since it seems to be a common default language that syntax highlighters support, but I could be wrong. Maybe others know more.